### DISCUSSION TITLE
Deploying Remix V2 App on Vercel

### DISCUSSION THREAD

**Discussion by hzhu** [remix-run/remix#7449] [Q&A] [answered]
Hello Remixooors 💿 ,

Has anyone been able to deploy a Remix V2 app on Vercel either from starting fresh with create-remix or upgrading from v1? A minimal demo repository would be appreciated. The only resource out there is the [V2 upgrade guide](https://remix.run/docs/en/main/start/v2#vercel-adapter), but doesn't seem to have enough information to get a deploy working on Vercel.

Here's the [repository](https://github.com/hzhu/remix-siwe) for a Remix V2 app I'm attempting to deploy on Vercel. I'm encountering the following error when I try to deploy on Vercel. Any suggestions would be greatly appreciated.

<img width="624" alt="Screenshot 2023-09-16 at 18 35 16" src="https://github.com/remix-run/remix/assets/1811365/5174f1fd-49b2-4fb2-bf3b-1de53b8c4a31">

**Comment by stormynight9** [comment]
Any updates on this issue?

**Comment by cill-i-am** [comment, marked as answer]
You will have to wait for Vercel to update their adapter V2. They manage it themselves.

**Comment by hzhu** [comment]
**UPDATE**: Vercel's adapter for V2 landed: https://www.npmjs.com/package/@vercel/remix
